### ISSUE TITLE
[Feature] Ability to rename documents

### ISSUE THREAD

**Issue by hyuri** [story-apps/starc#93]
Right now, we can't rename documents. And the name of text documents are automatically taken from first line of the document.

We should be able to rename documents, and documents should have an independent name not taken from the page itself.

**Comment by dimkanovikov** [comment]
Yes, agree with this point. The issue is to find proper visual solution (UI/UX) for this case...

**Comment by hyuri** [comment]
My suggestion would would simply giving us the option "Rename" in a context menu / right click in the scene navigator.
